package.json

```diff
@@ -117,6 +117,9 @@
     "vega-util": "^1.9.0",
     "yargs": "^13.2.2"
   },
+  "peerDependencies": {
+    "vega": "^5.1.0"
```

Comment:
This version of Vega-Lite doesn't work with 5.2?  If so, remove ^?

Reply:
Good point.